src/Kernel/xnu/osfmk/i386/cpuid.h

```diff
@@ -61,6 +61,13 @@
 #define _Bit(n)                 (1ULL << n)
 #define _HBit(n)                (1ULL << ((n)+32))
 
+/*
+ * This is so we don't have a global isAMD variable.
+ */
+#define CPUID_VEN_UNKNWON       0
```

Comment:
Also, typo here

Reply:
Will fix up soon, I have a local commit that hasn’t been pushed yet